.circleci/config.yml

```diff
@@ -0,0 +1,34 @@
+version: 2.1
+
+# Define the jobs we want to run for this project
+jobs:
+  build:
+    docker:
+      - image: pytorch/manylinux-builder:cpu
+    resource_class: 2xlarge+
+    steps:
+      - checkout
+      - run:
+          name: "Checking out PyTorch"
+          command: |   
+            git clone --depth 1 https://github.com/pytorch/pytorch /home/circleci/project/pytorch
+            export PACKAGE_TYPE=manywheel
+            export DESIRED_CUDA=cpu
+            export DESIRED_PYTHON=3.7m
+            export DESIRED_DEVTOOLSET=devtoolset7
+            export DOCKER_IMAGE=pytorch/manylinux-cpu # this may not be needed since we are inside the docker image 
+            export MAX_JOBS=$(nproc --ignore=2) 
+            bash manywheel/build_cpu.sh
+  test:
+    docker:
+      - image: circleci/python:3.7
+    steps:
+      - checkout
+      - run: echo "this is the test job"
```

Comment:
Do you really need a test workflow?
```suggestion

```

Reply:
This is a placeholder for test job (for now). We can implement this over the next PR.